x-pack/plugins/security/server/routes/authentication/index.ts

```diff
@@ -17,7 +17,12 @@ export function defineAuthenticationRoutes(params: RouteDefinitionParams) {
     defineSAMLRoutes(params);
   }
 
-  if (params.config.authc.sortedProviders.some(({ type }) => type === 'oidc')) {
+  if (
+    // this is not absolutely necessary, as OIDC provider isn't configured in serverless
+    // but should we add this just to make sure?
+    params.buildFlavor !== 'serverless' &&
```

Comment:
It's not necessary to make a build flavor check here because OIDC is not configured in serverless, but should have one anyway for good measure?